### PR TITLE
Plane.line_xsections: Document the behavior in a degenerate case

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: polliwog/plane/test_plane.py [hunk 1]
```diff
@@ -299,6 +299,26 @@ def test_line_plane_intersections():
...
+    plane = Plane(sample, normal)
+
+    # A line that lies in the x-z plane.
+    pts = np.array([[1.0, 0.0, -1.0]])
+    rays = np.array([[1.0, 0.0, 2.0]])
+
+    expected = np.array([[np.nan, np.nan, np.nan]])
+    intersections, is_intersecting = plane.line_xsections(pts, rays)
+    np.testing.assert_array_equal(intersections, expected)
+    np.testing.assert_array_equal(is_intersecting, [False])
```
Comment (paulmelnikow) [inline polliwog/plane/test_plane.py]: I'm not sure it's really worth adding this test, as it basically duplicates behavior from the previous test. If the decision is that this behavior is kept, I can add some comments to the method and the previous test, instead.

**Comment by paulmelnikow** [comment]
Hey @algrs, is this something you'd be able to weigh in on?
